rag/flow/tests/test_token_chunker_delimiter.py

```
@@ -0,0 +1,117 @@
+"""Regression test for token_size-mode delimiter handling in TokenChunker.
+
+The default ``delimiters`` a user supplies (e.g. ``["\\n"]``) are only honoured
+when they are backtick-wrapped; a bare delimiter makes
+``_compile_delimiter_pattern`` return ``""``, so ``TokenChunker._invoke`` falls
+through to ``naive_merge(payload, chunk_token_size, "")`` (token_chunker.py:323).
+
+Passing ``""`` discards the sentence-boundary preference that ``naive_merge``
+carries by default (``"\\n。；！？"``). Without a forced ``\\n`` break, the token
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF002)